xapi-backtrace.opam

```diff
@@ -6,11 +6,12 @@ bug-reports: "https://github.com/xapi-project/backtrace/issues"
 dev-repo: "git://github.com/xapi-project/backtrace.git"
 tags: [ "org:xapi-project" ]
 
-build: ["jbuilder" "build" "-p" name "-j" jobs]
+build: ["dune" "build" "-p" name "-j" jobs]
```

Comment:
We also need `--profile` release here.
The default is `dev` for dune.
See https://discuss.ocaml.org/t/dune-1-0-0-is-coming-soon-what-about-jbuilder-projects/2237 for info on migration.

Reply:
And we also need to check in the `dune-project` file